docs/content/ecs/commands.md

```diff
@@ -76,15 +76,14 @@ public class BuildSystem : ComponentSystem
             var targetEntityId = data.EntityIds[i];
 
             Builder.BuildWall.Request request = new Builder.BuildWall.CreateRequest
-            {
+            (
                 targetEntityId,
                 new BuildRequest
                 {
```

Comment:
I think these can also be `(` instead of `{`. We provide a constructor for requests